README.rst

```diff
@@ -95,38 +93,33 @@ directly, but instead be manually inspected. E.g.:
        fiel->feel, field, file, phial,
 
    Note the last comma! You need to use it, otherwise the last suggestion
-   will be discarded (see below for why). When there is more than one
+   will be discarded (for historic reasons). When there is more than one
    suggestion, an automatic fix is not possible and the best we can do is
    to give the user the file and line where the error occurred as well as
    the suggestions.
 
-3. Entry with one word, but with automatically fix disabled::
```

Comment:
Did we decide to stop officially supporting this?

Reply:
:shrug: But this is always telling me off for not adding one:
https://github.com/codespell-project/codespell/blob/master/codespell_lib/tests/test_dictionary.py#L110-L112

I suspect most of the time the risk of missing one and it becoming a reason is higher than the times we need to disable.

Also we now disable by just turning off a whole dictionary or skipping a specific word.

Reply:
It's possible people are using this feature, we should probably have some test that uses the functionality and makes sure it still works / disables autocorrection

Reply:
(i.e., just because internally we don't use it doesn't mean people's custom dictionaries don't. And it seems pretty easy to keep supporting this, assuming it currently does still work.)

Reply:
Yep true. Although by that argument we should possibly allow it via make check-dictionary too? We could also check for multiple words in the reason to catch missing accidental commas (until the multi-word thing comes in via #1657 )

Reply:
What would we need to change in `make check-dictionaries`? I don't see anything. The first steps don't check for commas (do they?), then the `pytest ...` line checks the internal ones, where we don't allow our internal dictionaries to have such entries.

Reply:
I meant to allow the pytest bit to work, while it's not required, it seems rather odd if someone can't just add their own dictionary to our file to easily run all our tests. Perhaps it wants an option like the aspell stuff to disable that test?

Reply:
> it seems rather odd if someone can't just add their own dictionary to our file to easily run all our tests. 

This suggests we should have a separate command like `codespell validate <filename>` that runs the Python-level validation on a given dictionary.

But even if we do that, we'd probably have switches for which things to check for. For our internal ones, we wouldn't allow the `, disabled because ...` because we don't need them, but for other people's dictionaries, we would for backward compatibility reasons.

So I'm still -1 on removing this information